### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,155 +1,17 @@
-# devextreme-react-template
+# DevExtreme React Template
 
-The DevExtreme React Template allows you to create an application with several views and a hierarchical menu (see the [live preview](https://devexpress.github.io/devextreme-react-template)).
+The DevExtreme React Template allows you to generate an application with a navigation menu and several sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). You can customize the generated application as you wish.
 
 ![DevExtreme-React-Template](https://user-images.githubusercontent.com/2280467/74529812-a8af1c80-4f39-11ea-96fb-efd94d19dbab.png)
 
-You can use [DevExtreme CLI](https://github.com/DevExpress/devextreme-cli) commands to modify, add, and delete views, configure the menu, and change themes.
+The DevExtreme React Template is created with [Create React App](https://github.com/facebook/create-react-app) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```
Comment (IlyaKhD) [inline README.md:7]:
```suggestion
The DevExtreme React Template is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```

How about using another link for Create React App ?

File: README.md [hunk 2]
```diff
@@ -1,155 +1,15 @@
-# devextreme-react-template
+# DevExtreme React Template
 
-The DevExtreme React Template allows you to create an application with several views and a hierarchical menu (see the [live preview](https://devexpress.github.io/devextreme-react-template)).
+The DevExtreme React Template is a React application with a navigation menu and several sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). This application is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```
Comment (dirk-pieterse) [inline README.md:3]:
```suggestion
The DevExtreme React Template is a React application with a navigation menu and sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). This application is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```

File: README.md [hunk 3]
```diff
@@ -1,155 +1,15 @@
...
 
 ![DevExtreme-React-Template](https://user-images.githubusercontent.com/2280467/74529812-a8af1c80-4f39-11ea-96fb-efd94d19dbab.png)
 
-You can use [DevExtreme CLI](https://github.com/DevExpress/devextreme-cli) commands to modify, add, and delete views, configure the menu, and change themes.
+## Getting Started
 
-The DevExtreme React Template is based on [DevExtreme React components](https://github.com/devexpress/devextreme-react) and generated with [Create React App](https://github.com/facebook/create-react-app).
+For full information about the DevExtreme React Template and how to customize it, refer to the following help topic: [Create a DevExtreme Application](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/).
```
Comment (dirk-pieterse) [inline README.md:9]:
```suggestion
For more information about the DevExtreme React Template and how to customize it, refer to the following help topic: [Create a DevExtreme Application](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/).
```

File: README.md [hunk 4]
```diff
@@ -1,155 +1,15 @@
...
-## <a name="license"></a>License
-
-Familiarize yourself with the
-[DevExtreme License](https://js.devexpress.com/Licensing/).
-[Free trial is available!](http://js.devexpress.com/Buy/)
+## License
 
 **DevExtreme React Template is released as a MIT-licensed (free and open-source) add-on to DevExtreme.**
+
+Familiarize yourself with the [DevExtreme License](https://js.devexpress.com/Licensing/). [Free trial is available](http://js.devexpress.com/Buy/).
```
Comment (dirk-pieterse) [inline README.md:15]:
```suggestion
More information about the [DevExtreme License](https://js.devexpress.com/Licensing/). [Free trial is available](http://js.devexpress.com/Buy/).
```

**Pull request merged**